package.json

```diff
@@ -329,8 +329,10 @@
     "chart.js": "^2.9.4",
     "circular-dependency-plugin": "^5.2.2",
     "color": "^3.2.1",
-    "concurrently": "^5.3.0",
-    "css-loader": "^5.2.7",
+
+    "concurrently": "^7.0.0",
+    "css-loader": "^6.5.1",
```

Comment:
Do we need to update css-loader too?

Reply:
No there is no need to upgrade the "ccs-loader" to a newer version, but the concurrently version has to be updated to the version 7.0.0 or higher to make the build process on windows 10.